static/js/containers/ProfileImage.js

```diff
@@ -36,6 +36,7 @@ class ProfileImage extends React.Component<*> {
     startPhotoEdit: (p: File) => void,
     updatePhotoEdit: (b: Blob) => void,
     useSmall?: boolean,
+    useMicro?: boolean,
```

Comment:
Rather than having multiple boolean fields for each type (which must/should all agree or we could end up in undefined behavior, e.g. `<ProfileImage useMicro useSmall />` is valid with this implementation), I'd suggest passing a single `size` prop that is typed as a union and determine whether you're using micro, small, medium off of that.

Reply:
yeah, that makes sense to me

Reply:
unfortunately it seems that because of the `connect` call the props for the `ProfileImage` aren't actually checked anywhere when the component is mounted...seeing if there's an easy way to work around that

Reply:
Would `flow-typed` defs for `react-redux` help flow infer that? https://github.com/flow-typed/flow-typed/blob/master/definitions/npm/react-redux_v5.x.x/flow_v0.63.0-/react-redux_v5.x.x.js

Reply:
yeah I added them, unfortunately that broke flow typings for a bunch of other things so I had to fiddle with it for a while. I just got it all typechecking now (haven't pushed the commits yet though)